### PR TITLE
Updating o-forms and adding unmasking

### PR DESCRIPTION
**Pull request by liamkeaton**
screen shot 2018-05-18 at 09 22 44
 🐿 v2.8.0

### CODE REVIEW

File: client/stylesheets/form.scss [hunk 1]
```diff
@@ -1,29 +1,58 @@
...
+
+	&__subheading {
+		@include oTypographySans($scale: 1);
+
+		@include oGridRespondTo(S) {
+			padding: 0 10px;
+		}
+	}
+
+	&-explination {
```
Comment (alexnaish) [inline client/stylesheets/form.scss]: `explanation`

File: views/form.html [hunk 2]
```diff
@@ -18,44 +26,81 @@ <h1 class="prospect-form__heading">Contact us for FT Group Subscription options<
...
 	</div>
-	<div class="prospect-form o-forms">
-		<label for="prospect-form__phone-number" class="prospect-form__label">Work phone number</label>
-		<small class="prospect-form__additional-info">5 to 15 characters (numbers only)</small>
-		<input name="phone" data-trackable="phone-number" type="text" id="prospect-form__phone-number" placeholder="Enter your phone number" class="prospect-form__text" required pattern="([0-9()+\- ])+" minlength="5" maxlength="15"></input>
-		<div class="prospect-form__errortext">Please enter a valid work phone number.</div>
+	{{/unless}}
+	<div class="o-forms">
+		{{#if isUnmasking}}
+		<label for="prospect-form__phone-number" class="o-forms__label">Phone number</label>
```
Comment (alexnaish) [inline views/form.html]: This `if` block could just be inside the label and change the text rather than dupe the element

**Review by alexnaish — COMMENTED** [review]
Would be good to see some tests too 👍

File: views/form.html (hunk 2 above)
Comment (liamkeaton) [reply views/form.html]: Thought it was easier to read like this? Don't mind changing though

File: client/stylesheets/form.scss (hunk 1 above)
Comment (liamkeaton) [reply client/stylesheets/form.scss]: Whoops

File: views/form.html (hunk 2 above)
Comment (alexnaish) [reply views/form.html]: Possibly, its just extra duplication

Comment (liamkeaton) [reply views/form.html]: 👍

**Review by alexnaish — APPROVED** [review]

**Pull request merged**